apps/frontend/src/libs/constants/navigation-items.constants.ts

```diff
@@ -0,0 +1,28 @@
+import { AppRoute } from "~/libs/enums/enums.js";
+import { type NavigationItem } from "~/libs/types/types.js";
+
+const sidebarItems: NavigationItem[] = [
```

Comment:
constants should be named in uppercase
```suggestion
const SIDEBAR_ITEMS: NavigationItem[] = [
```